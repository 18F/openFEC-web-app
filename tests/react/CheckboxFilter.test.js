const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const CheckboxFilter = require('../../static/js/legal/filters/CheckboxFilter');

it('renders checkbox', () => {
  const component = rendered.create(
    <CheckboxFilter key="ao_is_pending" name="ao_is_pending" label="Show only pending requests"
      checked={() => {}} handleChange={() => {}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
