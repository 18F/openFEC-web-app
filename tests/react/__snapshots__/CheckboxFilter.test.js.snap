// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders checkbox 1`] = `
<div
  className="filter"
>
  <div>
    <input
      checked={[Function]}
      id="ao_is_pending"
      name="ao_is_pending"
      onChange={[Function]}
      type="checkbox"
    />
    <label
      htmlFor="ao_is_pending"
    >
      Show only pending requests
    </label>
  </div>
</div>
`;
